grr/server/grr_response_server/bin/grrafana.py

```diff
@@ -46,20 +47,12 @@ class JSONRequest(werkzeug_wrappers_json.JSONMixin, werkzeug_wrappers.Request):
 class JSONResponse(werkzeug_wrappers_json.JSONMixin,
                    werkzeug_wrappers.Response):
 
-  def __init__(
-      self,
-      response=None,
-      status=None,
-      headers=None,
-      mimetype=None,
-      content_type=None,
-      direct_passthrough=False,
-  ):
-    try:
-      json_response = json.dumps(response)
-    except TypeError:
-      json_response = response
-    super().__init__(response=json_response, mimetype=JSON_MIME_TYPE)
+  def __init__(self, *args, **kwargs) -> None:
+    kwargs["mimetype"] = JSON_MIME_TYPE
+    response = kwargs.get("response")
+    if response and not isinstance(response, werkzeug_wsgi.ClosingIterator):
```

Comment:
Use `response is not None` instead of `response`. `if response` is `False` for `''` which would be a valid response.